src/graphics/aurora/borderquad.cpp

```diff
@@ -64,20 +64,21 @@ BorderQuad::BorderQuad(const Common::UString &edge, const Common::UString &corne
 	assert(!_edge.empty());
 }
 
-void BorderQuad::setPosition(float x, float y, float UNUSED(z)) {
+void BorderQuad::setPosition(float x, float y, float z) {
 	lockFrameIfVisible();
 
 	// Because of some graphic glitches with non-integer floats, i have to floor the values
 	_x = std::floor(x);
-	_y = std::floor(y);
+	_y = std::ceil(y);
```

Comment:
Could you explain *why* you're ceiling the value instead of flooring it? Update the comment above with your explanation, please.